lib/email_reply_parser.rb

```diff
@@ -166,6 +216,16 @@ def scan_line(line)
         end
       end
 
+      # Mark the current Fragment as a regards if regards are configured and
+      # the current line is empty and the Fragment starts with a common regards
+      # indicator.
+      if regards_regex && @fragment && line == EMPTY
```

Comment:
Not to keep repeating myself, but as discussed above I think we should fold this into the `signature` check.

But this does raise a question: should the signature customization be purely additive, or as a user of the library should I be able to clear out the default signature matchers? I'm inclined toward the latter.